File: DataServer/src/main/java/mediator/DataRequest.java
package mediator;

public class DataRequest
{
  private String Object;
  private String Type;
  private String Body;

  public DataRequest(String object, String type, String body)
  {
    this.Object = object;
    this.Type = type;
    this.Body = body;
  }

  public String getObject()
  {
    return Object;
  }
  
  public String getType()
  {
    return Type;
  }

  public String getBody()
  {
    return Body;
  }
}

File: WebAPI/IncWebAPI/DataMediator/DataRequest.cs
using System;

namespace WebAPI.DataMediator 
{
    public class DataRequest 
    {
        public string Object { get; }    
        public string Type { get; }
        public string Body { get; }

        public DataRequest(string obj, string type, string body)
        {
            Object = obj;
            Type = type;
            Body = body;
        }
    }
}